CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
Looks like my edit trimmed some trailing whitespace. Let me know if this is a problem and I'll fix.

Reply:
No worries